src/test/java/seedu/address/model/room/RoomIntegrationTest.java

```diff
@@ -0,0 +1,32 @@
+package seedu.address.model.room;
```

Comment:
Good! We will need to add test codes for patient addition for editRoom command too.